clients/common/utility.cpp

```diff
@@ -79,22 +79,22 @@ int type2int<hipblasDoubleComplex>(hipblasDoubleComplex val)
 std::string hipblas_exepath()
 {
 #ifdef WIN32
-    std::vector<wchar_t> result(MAX_PATH + 1);
+    std::vector<TCHAR> result(MAX_PATH + 1);
     // Ensure result is large enough to accomodate the path
     for(;;)
     {
-        auto length = GetModuleFileNameW(nullptr, result.data(), result.size());
+        auto length = GetModuleFileNameA(nullptr, result.data(), result.size());
         if(length < result.size() - 1)
         {
             result.resize(length);
-            result.shrink_to_fit();
```

Comment:
You'll want the resize length + 1 as Cory noted it is actually not counting the null character for this success return value.